pages/updates/index.vue

```diff
@@ -0,0 +1,761 @@
+<template>
+	<main>
+		<article id="5e3ea028-0697-4f95-89e4-0ed44d5f8b15" class="page sans">
```

Comment:
what is that id for or how was it created?

Reply:
That is generated by Notion which I believe the update pages were exported from.